键盘/陈奕迅/胶囊65/readme.md

```diff
@@ -0,0 +1,18 @@
+# capsule65
```

Comment:
This file has an invalid path, and should be `keyboards/eason/capsule65/readme.md`.